docs/containers-storage.conf.5.md

```diff
@@ -115,7 +115,13 @@ Specifies extra mkfs arguments to be used when creating the base device.
 
 **mountopt**=""
 
-Specifies extra mount options used when mounting the thin devices.
+Default options to be used to mount container images.  Suggested value "nodev".
+
+**ostree_repo=""**
+  Tell storage drivers to use the specified OSTree repository.  Some storage drivers, such as overlay, might use
```

Comment:
Looks like we dropped some text here.

Reply:
This shouldn't be in the `storage.options.thinpool` section.

Reply:
Removed from my patch set.